asab/library/providers/filesystem.py

```diff
@@ -37,8 +37,9 @@ async def list(self, path, recursive=True):
 		)
 
 		# Remove library path from the beginning of file names
-		library_path_to_replace = "{}/".format(os.path.abspath(self.LibraryBaseDir))
-		file_names_list = [i.replace(library_path_to_replace, '', 1) for i in file_names]
+		library_path_to_replace = "[string_start]{}/".format(os.path.abspath(self.LibraryBaseDir))
+		labeled_file_names = ["[string_start]" + i for i in file_names]
+		file_names_list = [i.replace(library_path_to_replace, '') for i in labeled_file_names]
```

Comment:
Nope, that's not a good solution. Use `.find() == 0` ...

Reply:
What about string slicing?
```
if str.startswith(path_to_replace):
    str = str[len(path_to_replace):]
```

Reply:
Yes, that's ok ... but implement also `else` with some kind of failure ... or `asset str.startswith(path_to_replace)`